src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -20,6 +21,8 @@ public PullRequestCreationView()
 
             this.WhenActivated(d =>
             {
+                d(this.BindCommand(ViewModel, vm => vm.Cancel, v => v.cancelButton));
```

Comment:
Ok, so are we Officially going to prefer XAML bindings over ReactiveUI bindings from here on? This was like this just because it's what the other views do.